packages/story-editor/src/components/canvas/utils/useFullbleedMediaAsBackground.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import isTargetCoveringContainer from '../../../utils/isTargetCoveringContainer';
+import { useStory, useCanvas } from '../../../app';
+import { MEDIA_ELEMENT_TYPES } from '../../../elements';
+
+function useFullbleedMediaAsBackground({ selectedElement }) {
+  const { setBackgroundElement, isDefaultBackground } = useStory((state) => ({
+    setBackgroundElement: state.actions.setBackgroundElement,
+    isDefaultBackground:
+      state.state.currentPage?.elements[0]?.isDefaultBackground,
+  }));
+  const { fullbleedContainer } = useCanvas(
+    ({ state: { fullbleedContainer } }) => ({
+      fullbleedContainer,
+    })
+  );
+
+  const handleFullbleedMediaAsBackground = (target) => {
+    if (!isDefaultBackground) {
+      return false;
+    }
```

Comment:
This confused me a lot for several minutes. I thought it checked it the selected element or the target was a default background, which didn't make a lot of sense. But it reflects whether the actual background on the current page is a default background and thus not a media element. Can we rename `isDefaultBackground` a bit to reflect this? Maybe just `pageHasDefaultBackground` or something like that?

Reply:
@barklund `isDefaultBackground` is something that's used across the whole codebase currently to check if something is media or the default shape. This prop name came already from https://github.com/GoogleForCreators/web-stories-wp/pull/2387 if I remember correctly.

Probably just leaving a comment for now is sufficient, e.g. `// Return false if it's not background media`.

If you feel like changing it generally, we should create a code quality issue for it instead.

Reply:
I just realized that maybe you meant just changing the name in the selector above and not generally in the code, ignore my previous comment in this case! 🐶